SUPPORT.md

```diff
@@ -0,0 +1,28 @@
+# Support
+
+## Thoth wiki
+The Thoth team actively contributes to its [Wiki](../../wiki), if you have a question about book metadata we might have an entry about it.
+
+Our wiki also includes the [Thoth User Manual](../../wiki/Thoth-User-Manual).
+
+## Asking quality questions
+> This section has been borrowed from Titus Wormer's [remarkjs support document](https://github.com/remarkjs/.github/blob/main/support.md)
+
+Questions can go to [GitHub discussions][https://github.com/remarkjs/remark/discussions].
+
+Help us help you!
+Spend time framing questions and add links and resources.
+Spending the extra time up front can help save everyone time in the long run.
+Here are some tips:
+
+*   [Talk to a duck][https://rubberduckdebugging.com]!
+*   Don’t fall for the [XY problem][https://meta.stackexchange.com/questions/66377/what-is-the-xy-problem/66378#66378]
```

Comment:
Links work now but don't display cleanly - fix Markdown syntax (wrong type of brackets)

Reply:
Actually, the GitHub discussions link goes to the wrong repository!

Reply:
Oh dear. Sorry - fixed now